Changelog.md

```diff
@@ -6,6 +6,8 @@ Language Features:
 
 
 Compiler Features:
+ * Code Generator: Insert helper functions for panic codes instead of inlining unconditionally. This can reduce costs if many panic (checks) are inserted,
+   but can increase costs where few panics are used.
```

Comment:
Please double check this wording.

Reply:
Maybe `Call helper functions for panic codes instead ...`?

Reply:
Or `Generate helper functions ...`?

But looks fine overall.